### PR TITLE
(BKR-510) Support for EC2 Netscaler provisioning

### PR DESCRIPTION
**Pull request by LuvCurves**

### CODE REVIEW

**Comment by LuvCurves** [comment]
retest this please

**Comment by LuvCurves** [comment]
Smoketest failing on Ubuntu15.04 because it can't find the scooter gem.  Not a problem with the PR

File: lib/beaker/hypervisor/aws_sdk.rb [hunk 1]
```diff
@@ -549,9 +549,9 @@ def etc_hosts_entry(host, interface = :ip)
     # @return [void]
     # @api private
     def configure_hosts
-      @hosts.each do |host|
-        next if host['platform'] =~ /f5/
-        host_entries = @hosts.map do |h|
+      non_netdev_hosts = @hosts.select{ |h| !(h['platform'] =~ /f5|netscaler/) }
```
Comment (colinPL) [inline lib/beaker/hypervisor/aws_sdk.rb]: This is using `f5` and not `f5-` like other statements above.

File: lib/beaker/hypervisor/aws_sdk.rb [hunk 2]
```diff
@@ -464,16 +464,16 @@ def wait_for_status(status, instances, &block)
...
+    # @note if any host is an netdev one, these checks will happen once across all
     #   of the hosts, and then we'll exit
     #
     # @return [void]
     # @api private
-    def wait_for_status_f5()
+    def wait_for_status_netdev()
       @hosts.each do |host|
-        if host['platform'] =~ /f5/
+        if host['platform'] =~ /f5-|netscaler/
```
Comment (colinPL) [inline lib/beaker/hypervisor/aws_sdk.rb]: Should `cumulus` be included here?

Comment (LuvCurves) [inline lib/beaker/hypervisor/aws_sdk.rb]: No, cumulus works just fine.  These NetDev types actually use 'puppet device' and don't have puppet agents like Cumulus, Arista & Cisco

**Pull request merged**